pkg/kubectl/cmd/util/BUILD

```diff
@@ -17,9 +14,7 @@ go_library(
         "printing.go",
         "shortcut_restmapper.go",
     ],
-    visibility = [
-        "//build/visible_to:pkg_kubectl_cmd_util_CONSUMERS",
```

Comment:
Please don't undo this. This is working to snip dependencies to move kubectl out of the repo